web_app.py

```
@@ -2538,39 +2552,54 @@ def _chat_stream_payload(
+        def _map_error(event: Any):
+            if type(event).__name__ != "RunErrorEvent":
+                return None
+            # #1452/#1465：无 typed status 的 stream error 不洗成瞬断
+            return transport_failure_unavailable(
+                run_error_event_failure(getattr(event, "content", None)),
+                attempts=1,
+                exhausted=False,
+            )
+
+        def _is_activity(event: Any) -> bool:
+            # 空转活动：RunContent 有正文，或带 reasoning 增量（活动 ≠ 已呈现正文）
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)